### PR TITLE
2022-05-02-Raccontiamo le-esperienze-di-sviluppo-della-Pubblica-Ammin…

### PR DESCRIPTION
**Pull request by Francescopog**
…istrazione.md

## Description

@bfabio 

This PR tackles:

* ...
* ...
* ...

In particular, the ...

## Checklist

* [ ] I followed the indications in [CONTRIBUTING.md](https://github.com/italia/developers.italia.it/blob/master/CONTRIBUTING.md)
* [ ] The documentation related to the proposed change has been updated accordingly (also comments in code).
* [ ] Have you written new tests for your core changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?
* [ ] Ready for review! :rocket:

## Fixes

Fixes #

### CODE REVIEW

**Comment by developers-italia-droid** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>There is a problem with the commit message
> 2022-05-02-Raccontiamo le-esperienze-di-sviluppo-della-Pubblica-Amministrazione.md
- subject may not be empty
- type may not be empty</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 22195dbc89a3fab2012187d4ce7757f29c650b67
</p>

File: _posts/it/2022-05-02-Raccontiamo le-esperienze-di-sviluppo-della-Pubblica-Amministrazione.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+### Contribuisci
+
+Questo è solo l’inizio. Nel rispetto della filosofia *open* vogliamo usare il nuovo blog come spazio dove dare voce e amplificare il lavoro di chiunque stia sviluppando un progetto open source per la PA particolarmente interessante o di valore. Stiamo cercando persone che fanno software e hanno voglia di parlarne. Ti va di salire a bordo?
+
+### Cosa puoi fare
+
+Visita il [nuovo blog di Developers Italia](https://medium.com/developers-italia/)
+
+Mandaci la tua idea scrivendo a contatti@developers.italia.it con una breve presentazione di cosa fai, per quale ente pubblico lavori e di quale esperienza di sviluppo intendi parlare.
```
Comment (bfabio) [inline _posts/it/2022-05-02-Raccontiamo le-esperienze-di-sviluppo-della-Pubblica-Amministrazione.md:29]:
```suggestion
Mandaci la tua idea scrivendo a [contatti@developers.italia.it](mailto:contatti@developers.italia.it) con una breve presentazione di cosa fai, per quale ente pubblico lavori e di quale esperienza di sviluppo intendi parlare.
```

File: _posts/it/2022-05-02-Raccontiamo le-esperienze-di-sviluppo-della-Pubblica-Amministrazione.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+locale: it
+---
+Il software si fa, ma non si dice. Anzi, si, parliamone! Progettare e sviluppare software open source per la Pubblica Amministrazione vuol dire lavorare e contribuire a un bene pubblico di inestimabile valore per la collettività. 
+
+Developers Italia intende sempre di più proporsi come la “voce” di chi lavora nello sviluppo software della Pubblica Amministrazione. Non solo per aiutare tutti coloro che lavorano nel campo ad accedere a strumenti aperti e usare soluzioni in riuso, ma anche per diffondere buone pratiche e raccontare le storie, i “dietro le quinte” di chi lavora sul codice di quei servizi che usiamo ogni giorno. 
+
+Per questo motivo nasce la **[nuova pubblicazione](https://medium.com/developers-italia)** dedicata a chi lavora allo sviluppo di software che stanno alla base dei servizi pubblici digitali. Basata su Medium ma disponibile anche sulla piattaforma open source Noblogo, la pubblicazione ospiterà i racconti di chi ha sviluppato e/o implementato siti web, servizi e applicazioni della Pubblica Amministrazione. Sarà uno spazio aperto a tutti coloro che vogliono raccontare una storia di sviluppo della e per la PA.
```
Comment (bfabio) [inline _posts/it/2022-05-02-Raccontiamo le-esperienze-di-sviluppo-della-Pubblica-Amministrazione.md:16]:
```suggestion
Per questo motivo nasce la **[nuova pubblicazione](https://medium.com/developers-italia)** dedicata a chi lavora allo sviluppo di software che stanno alla base dei servizi pubblici digitali. Basata su Medium ma disponibile anche sulla piattaforma open source NoBlogo, la pubblicazione ospiterà i racconti di chi ha sviluppato e/o implementato siti web, servizi e applicazioni della Pubblica Amministrazione. Sarà uno spazio aperto a tutti coloro che vogliono raccontare una storia di sviluppo della e per la PA.
```

**Comment by bfabio** [comment]
@Francescopog the URL is generated from the name of file: let's use only lower case letter and dashes (`-`), no spaces or capitalizations

**Pull request merged**